src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotationFactory.java

```diff
@@ -1320,9 +1347,16 @@ private static GencodeGtfExonFeature getExonWithinSpliceSiteWindow( final Varian
                                                                         final int spliceSiteVariantWindowBases ) {
         GencodeGtfExonFeature spliceSiteExon = null;
 
+        final int varStart = FuncotatorUtils.getIndelAdjustedAlleleChangeStartPosition(variant);
+        final int varEnd = variant.getEnd();
+
         for ( final GencodeGtfExonFeature exon : transcript.getExons() ) {
-            if ((Math.abs(exon.getStart() - variant.getStart()) <= spliceSiteVariantWindowBases) ||
-                    (Math.abs(exon.getEnd() - variant.getStart()) <= spliceSiteVariantWindowBases)) {
+            // Check the start and end of the variant to see if it overlaps with either end of the exon:
+            if ((Math.abs(exon.getStart() - varStart) <= spliceSiteVariantWindowBases) ||
```

Comment:
Can you use something like `SimpleInterval.overlapsWithMargin()` here?

Reply:
Yes.  Fixed!